packages/SwingSet/src/netstring.js

```diff
@@ -49,14 +51,15 @@ export function decode(data) {
     const size = parseInt(sizeString, 10);
     if (!(size > -1)) {
       // reject NaN, all negative numbers
-      throw Error(`unparseable size '${sizeString}', should be integer`);
+      assert.fail(X`unparseable size ${q(sizeString)}, should be integer`);
```

Comment:
probably doesn't need `q()`

Reply:
This was also due to trying to satisfy a test.
https://github.com/Agoric/agoric-sdk/blob/17d7df6ee06eb5c340500bb5582f985c2993ab19/packages/SwingSet/test/test-netstring.js#L109

I'll fix both this and the test.